internal/handlers/shared/msg_dropdatabase.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jackc/pgx/v4"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// MsgDropDatabase removes the current database.
+func (h *Handler) MsgDropDatabase(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	m := document.Map()
+	db := m["$db"].(string)
+
+	sql := fmt.Sprintf(`DROP SCHEMA %s CASCADE`, pgx.Identifier{db}.Sanitize())
+
+	_, _ = h.pgPool.Exec(ctx, sql)
```

Comment:
Will want to check for errors etc here

Reply:
Just return a lazy error if something happened